doc/sailfish-webview-webviewexperimental.qdoc

```diff
@@ -0,0 +1,181 @@
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+\page webview-experimental.html
+\title Sailfish WebView Experimental Attributes
+
+Each one of the properties, signals, and methods provided by Sailfish WebView
+falls into one of the three following categories:
+\list
+\li Stable attributes, as per the WebView type documentation
+\li \l{Sailfish WebView Additional Attributes}{Additional attributes}
+\li \l{Sailfish WebView Experimental Attributes}{Experimental attributes}
+\endlist
+
+This page documents the experimental attributes of the Sailfish WebView.
+These experimental attributes are highly unlikely to be used by vast majority
+of application developers, however they allow developers great control
+over the way content is displayed in their application.
+
+We reserve the right to modify or remove any of these attributes in future
+versions of the \c{Sailfish.WebView} import.
+
+\section1 Properties
+
+\section2 WebView::contentRect
+
+\c{QRectF}-type read-only property.
+
+The bounding box rectangle defined by the loaded content being displayed in
+the viewport of the web view.
+
+The loaded page content is likely to fill an area much larger than the
+viewport of the webview (thus requiring the user to scroll the viewport in
+order to see content further down the page, etc).
+
+This property provides the bounding box rectangle of the current viewport
+onto the content, in CSS pixels (that is, not yet scaled by the device
+resolution factor).  The x and y position of the content rectangle describes
+the top-left pixel of the content being displayed in the viewport.  The
+width and height of the content rectangle can be combined with the x and y
+positions to determine the top-right, bottom-left, and bottom-right pixels
+of the content being displayed in the viewport.
+
+Note that the contentWidth and contentHeight will differ from the
+viewport width and viewport height only if some zoom-factor is being applied
+to the viewport, or if the content is smaller than the viewport.
+
+For more information about these concepts, please see the related Mozilla
+documentation about
+\l{https://developer.mozilla.org/en-US/docs/Glossary/CSS_pixel}{CSS pixels}
+and
+\l{https://developer.mozilla.org/en-US/docs/Web/CSS/Viewport_concepts}{viewports}.
+
+See also: \l{WebView::contentWidth}, \l{WebView::contentHeight},
+\l{WebView::viewportWidth}, \l{WebView::viewportHeight},
+\l{WebView::scrollableSize}
+
+\section2 WebView::contentWidth
+
+\c{qreal}-type read-only property.
+
+The width of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::contentHeight
+
+\c{qreal}-type read-only property.
+
+The height of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
```

Comment:
Similarly, it'd be good to know the units here too.

Reply:
done